### PR TITLE
tests: undefined variable "self" should be void 0 in the vm

### PR DESCRIPTION
**Pull request by watilde**
To fix the error on the CI like the following:
https://travis-ci.org/andyearnshaw/Intl.js/builds/137850735

### CODE REVIEW

**Comment by watilde** [comment]
Hmmm something is happening on the CI, but I can't reproduce it on my local...

File: tests/noderunner.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ function runTest(testPath) {
     var content,
         context = LIBS.vm.createContext({});
 
+    context.global = context
```
Comment (caridy) [inline tests/noderunner.js]: this should do the trick IMO, just missing a `;`

**Comment by watilde** [comment]
Hmm the reason of the error is `ReferenceError: self is not defined` in the vm 💭 
```
evalmachine.<anonymous>:1022
  var selfGlobal = typeof global === "undefined" ? self : global;
                                                   ^
ReferenceError: self is not defined
    at evalmachine.<anonymous>:1022:52
    at evalmachine.<anonymous>:4:26
    at evalmachine.<anonymous>:5:2
    at Object.<anonymous> (/home/travis/build/andyearnshaw/Intl.js/tests/polyfilling.js:24:8)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
cache.2
store build cache
0.00s
2.14snothing changed, not updating cache
Done. Your build exited with 1.
```

**Comment by watilde** [comment]
Ok it was fixed. undefined variable "self" should be void 0 in the vm.

**Comment by caridy** [comment]
thanks @watilde

**Pull request merged**